projects/managertab.php

```diff
@@ -43,7 +43,7 @@
 			<input name="pid" type="hidden" value="<?php echo $pid; ?>" />
 			<input name="tabindex" type="hidden" value="1" />
 			<input name="emode" type="hidden" value="1" />
-			<button name="projsubmit" type="submit">
+			<button name="projsubmit" type="submit" value="Add to Manager List">
```

Comment:
This is the only change from the cherry pick. The rest of these changes must be coming from the master branch back into hotfix???

Reply:
@Atticus29 , I noticed this problem when I tried to merge something else into hotfix, too...